test/repl_test.js

```diff
@@ -13,6 +13,8 @@ var colors = require('colors')
 var payload = ''
 var count = 0
 
+var PADDING = 19
```

Comment:
in a test file, this meta info should be kept at the end : 

```
## $$TEST_METHOD$$
## test_hello_world()
```

So, from beginning $ to the space before `test_hello_world`, there are 19 characters, which serves as a padding to extract the method name.

The method name is required to run the tests in REPL.